WebRoot/js/zimbraMail/share/model/ZmMetaData.js

```diff
@@ -105,6 +105,14 @@ function(section, data, batchCommand, callback, errorCallback, sensitive) {
 		if (AjxUtil.isObject(d)) {
 			d = ZmSetting.serialize(d, ZmSetting.D_HASH);
 		}
+
+		// ToDo:- Remove this code after ZCS-9866 is closed.
+		if (i === "zimbraPrefFoldersExpanded" && (d === "" || d == null)) {
+			// Adding hardcoded value 3:true to always send non empty value of zimbraPrefFoldersExpanded
+			// bacause as of now backend doesn't support empty values of key.
+			d = '3:true';
```

Comment:
Add to the comment above: 3 is drafts folder. does not have subfolders. hence this is a noop

Reply:
sure